oat-sa/extension-tao-itemqti
# Fix/aut 445/do not save empty custom css

https://oat-sa.atlassian.net/browse/AUT-445

How to test:
1. Save item with custom styles (use Style editor) - item XML should contain `<stylesheet href="style/custom/tao-user-styles.css" type="text/css" media="all" title="" />` and in item package should be `tao-user-styles.css` file
2. Clear all custom styles and save - item XML should NOT contain `<stylesheet href="style/custom/tao-user-styles.css" type="text/css" media="all" title="" />` and in item package should NOT be `tao-user-styles.css` file
